src/gui/gui_element_base.hpp

```diff
@@ -118,13 +118,11 @@ class element_base {
 
 template<typename T>
 inline T retrieve(sys::state& state, element_base* parent) {
+	Cyto::Any payload = T{};
```

Comment:
I don't understand why you made this change. It will still function the same, of course, but it is just slightly slower in the case where parent is null